CAIPs/caip-282.md

```diff
@@ -0,0 +1,198 @@
+---
+caip: 282
+title: Browser Wallet Messaging Interface
+author: Pedro Gomes (@pedrouid)
+discussions-to: https://github.com/ChainAgnostic/CAIPs/pull/282
+status: Draft
+type: Standard
+created: 2023-05-30
+requires: 2, 10, 25, 27, 222, 275
+---
+
+## Simple Summary
+
+Standardized messaging for wallet interface in browser environments.
```

Comment:
```suggestion
Standardized messaging interface for browser wallets.
```

Reply:
This depends on the definition of a browser wallet... personally I classify a browser wallet as a wallet that is integrated into the browser either natively or as an extension

Would like to avoid limiting this standard to only those wallets since this should support any wallet that can access the browser environment including embedded wallets, cloud wallets, iframe wallets and remote wallets